openeo/rest/datacube.py

```diff
@@ -810,6 +821,11 @@ def _get_geometry_argument(
     ) -> Union[dict, Parameter, PGNode]:
         """
         Convert input to a geometry as "geojson" subtype object.
+
+        :param crs: data structure that encodes a CRS, typically just an int or string value.
+            If the ``pyproj`` library is available, everything supported by it is allowed.
```

Comment:
I would not mention this pyproj remark everywhere, and just move that to normalize_crs's docs

Reply:
Done, just referring to normalize_crs now.